cl/lib/model_helpers.py

```diff
@@ -1,7 +1,7 @@
 import contextlib
 import os
 import re
-from typing import Optional
+from typing import Callable, Dict, List, Optional
```

Comment:
Importing `Dict` and `List` from typing is redundant. We can use built-in types (`dict` and `list`):

```suggestion
from typing import Callable, Optional
```